package.json

```diff
@@ -4,12 +4,17 @@
 	"description": "Creating fixtures based on zod schemas",
 	"packageManager": "npm@9.5.0",
 	"scripts": {
-		"start": "vite",
-		"test": "vitest --ui",
-		"test:coverage": "vitest run --coverage",
-		"build": "vite build",
+		"start": "vite -c vite.lib.config.ts",
```

Comment:
Somehow vitpress picks up the root `vite.config.ts` file and this is causing issues.
I don't see another way to prevent this, I'll have to try some thing out later to revert these changes.